apps/mark-scan/backend/src/custom-paper-handler/state_machine.ts

```diff
@@ -502,8 +532,20 @@ export function buildMachine(
             accepting_paper: {
               invoke: [pollPaperStatus(), pollAuthStatus()],
               on: {
-                PAPER_READY_TO_LOAD: 'loading_paper',
-                PAPER_PARKED: 'waiting_for_ballot_data',
+                PAPER_READY_TO_LOAD: [
+                  {
+                    target: 'loading_new_sheet',
+                    cond: isBallotReinsertionEnabled,
+                  },
+                  'loading_paper',
+                ],
+                PAPER_PARKED: [
+                  {
+                    target: 'not_accepting_paper',
+                    cond: isBallotReinsertionEnabled,
+                  },
```

Comment:
Yeah, I was tempted to remove the `PAPER_PARKED` trigger from this state altogether, since it didn't seem like reachable code. Left it for now, in case it was originally handling an obscure edge case - do you have any context on the intent behind the original code?

Reply:
Ah I remember now, this handles if paper is left parked from a previous session. That really only makes sense in the development flow when the state machine needs to be restarted. I think this is redundant with `PAPER_PARKED: 'eject_to_front'` in `not_accepting_paper`. Can we delete `PAPER_PARKED` in `accepting_paper` if it doesn't break too many tests?

Reply:
+1! Will clean up

Reply:
✅  [e9189d9](https://github.com/votingworks/vxsuite/pull/5024/commits/e9189d99360f07677c2e8e25f3765d82469f197b)